server/server/settings.py

```diff
@@ -90,8 +90,15 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'mysql.connector.django',
+        'NAME': 'test160',
+        'USER': 'root',
+        'PASSWORD': 'secretpassword',
```

Comment:
In production, you can't have these values be public. Even if it's on `localhost`, it's on the localhost of your server.